packages/gatsby-codemods/src/transforms/__testfixtures__/global-graphql-calls/import-default.output.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable */
-import Gatsby, { graphql } from "gatsby";
+import Gatsby, { graphql } from "gatsby"
```

Comment:
Hmm maybe we should have a `.prettierignore` that skips anything in `__testfixtures__`

Reply:
I tried doing this but it seems to be breaking more tests or I'm doing something wrong :smile: 

Will try to get them fixed tomorrow.